cmd/jujud/agent/bootstrap.go

```diff
@@ -217,7 +218,10 @@ func (c *BootstrapCommand) Run(_ *cmd.Context) error {
 		if isCAAS {
 			// For CAAS, the agent-version in controller config should
 			// always equals to current juju version.
-			return errors.NotSupportedf("desired juju version for k8s controllers")
+			return errors.NotValidf(
```

Comment:
this will append "not valid" to the error message and it will read incorrectly

Reply:
fixed